browser/src/map/handler/Map.Keyboard.js

```diff
@@ -140,9 +140,106 @@ L.Map.Keyboard = L.Handler.extend({
 	},
 
 	keyCodes: {
+
+		BACKSPACE:8,
+		TAB:      9,
+		enter:   13, 
+		SPACE :   32,
 		pageUp:   33,
 		pageDown: 34,
-		enter:    13
+		SHIFT:    16, // shift		: UNKOWN
+		CTRL:     17, // ctrl		: UNKOWN
+		ALT:      18, // alt		: UNKOWN
+		pause:    19, // pause/break	: UNKOWN
+		capslock: 20, // caps lock	: UNKOWN, 
+		END:      35,
+		HOME:     36,
+		LEFT:     37,
+		UP:       38,
+		RIGHT:    39,
+		DOWN:     40,
+		INSERT:   45,
+		DELETE:   46,
+		
+		NUM0:     [48,96],
```

Comment:
There were two values for `NUM0 `in `keymap`. I have clubbed them together like this in `keyCodes`, Is my approach right?

Reply:
one for safari/iOS, other for other systems? 

If it's needed and true lets keep it that way :)

Reply:
> one for safari/iOS, other for other systems?

yes. Okay will keep it